src/edu/ucsd/library/xdre/harvesting/CilHarvesting.java

```diff
@@ -646,7 +649,7 @@ private String formatRelatedPublications(String val) {
      * Extract File use from source file type
      * if value = Jpeg|OME_tif THEN image-source
      * if value = Gzip|Zip|Mrc THEN data-service
-     * if value = Flv THEN video-source
+     * if value = mp4 THEN video-source
```

Comment:
Is `Flv` still possible? The `else if` on line 662 makes it look like we need to still check for both. If so, should this be `Flv|mp4`?

Reply:
@mcritchlow I think we should just try to leave the old format `Flv` there since we will never know whether there are still any such files or not, though the instruction is updated with the mp4 format. But I am fine if you don't want to have it.

Reply:
Whatever you think  is best. What stood out to me was the instruction here being switched to only mp4, but the code itself still checking for `Flv`. So I was just wondering if we should be consistent. Given the history with this project so far, your choice to keep the check around makes a lot of sense to me :)

Reply:
Okay, I've added a note for it since it's not in the mapping instruction.